bootstrap_cfn/config.py

```diff
@@ -225,6 +227,17 @@ def elb(self):
                     listener.update(ssl_template)
 
 
+            elb_sg = template.pop('DefaultELBSecurityGroup')
+            if 'security_groups' in elb:
+                for sg_name, sg in elb['security_groups'].items():
+                    new_sg = deepcopy(elb_sg)
+                    new_sg['Properties']['SecurityGroupIngress'] = sg
+                    elb_sgs[sg_name + safe_name] = new_sg
```

Comment:
The above commend applies more to line 238 `['DefaultSG' + safe_name]`.

For this one: it's slightly unexpected that the SG name you put in the yaml isn't the SG name you'd put elsewhere to reference it...